### ISSUE TITLE
Can OpenFace be used for commercial purposes?

### ISSUE THREAD

**Issue by florinmatei** [TadasBaltrusaitis/OpenFace#113]
I've read Copyright.txt but is all written in the lawyer terms which I can't confidently say I fully understand. In the layman terms can OpenFace be used/integrated with commercial products?

**Comment by PVNathan** [comment]
Yes, I would like to know that too. Also if at all someone is interested to use it in a commercial product, is there a possibility to maybe buy the copyright? 
Thanks

**Comment by TadasBaltrusaitis** [comment]
For commercial licensing of OpenFace please contact Cindy Chepanoske - cchepano@andrew.cmu.edu

Thanks,
Tadas

**Comment by PVNathan** [comment]
Thanks

**Comment by doczoidberg** [comment]
Unfortunately I get no response on that mail address.

**Comment by dheerajmpai** [comment]
screw it
